feat/gene/gene.go

```diff
@@ -220,39 +220,43 @@ func (t *CodingTranscript) Orientation() feat.Orientation {
 
 // UTR5start returns the start of the 5'UTR relative to the transcript.
 func (t *CodingTranscript) UTR5start() int {
-	if t.Orient == feat.Forward {
+	ori, _ := feat.BaseOrientationOf(t)
+	if ori == feat.Forward {
 		return 0
-	} else if t.Orient == feat.Reverse {
+	} else if ori == feat.Reverse {
 		return t.CDSend
```

Comment:
Nope they should not be mapped through `BasePositionOf`.

You are right a switch is nicer. I'll make the change.